server/application-server/src/main/java/de/tum/in/www1/hephaestus/gitprovider/label/Label.java

```diff
@@ -47,6 +48,10 @@ public class Label {
     @JoinColumn(name = "repository_id")
     @ToString.Exclude
     private Repository repository;
+    
+    @ManyToMany(mappedBy = "labels")
```

Comment:
mappedBy="teams"

Reply:
`teams` should be correct.
It refers to the attribute name in the `Team`-Entity, which contains the relationship.